docs/reference/execution-providers/TensorRT-ExecutionProvider.md

```diff
@@ -22,7 +22,7 @@ With the TensorRT execution provider, the ONNX Runtime delivers better inferenci
 
 See [Build instructions](../../how-to/build/eps.md#tensorrt).
 
-The TensorRT execution provider for ONNX Runtime is built and tested with TensorRT 7.1.3.4.
+The TensorRT execution provider for ONNX Runtime is built and tested with TensorRT 7.2.2.
```

Comment:
both 7.1.3.4 and 7.2.2 can work.

Reply:
Do we have tests with 7.1.3.4? Is it normally expected that this is backwards compatible or should it be explicitly called out?

Reply:
We don't have CI build for TRT7.1 after we moved to TRT7.2. synced with @jywu-msft offline and we probably need such CI build for TRT7.1

Reply:
@jywu-msft @faxu Please help to merge the PR. It seems I am not authorized to merge.